cli_tools/common/image/importer/api_inflater_test.go

```diff
@@ -151,7 +152,7 @@ func TestAPIInflater_getCalculateChecksumWorkflow(t *testing.T) {
 		ExecutionID:  "1234",
 		NoExternalIP: false,
 		WorkflowDir:  daisyWorkflows,
-	}, nil, &storage.Client{}, logging.NewToolLogger(t.Name()), true)
+	}, nil, &storage.Client{}, logging.NewToolLogger(t.Name()), imagefile.Metadata{}, true)
```

Comment:
in my opinion there should be one testcase to validate whether the "sourceType" is populated correctly

Reply:
I wasn't able to mock the storage client appropriately to test the PD fields, but instead I added a check in `inflater_test.go` to make sure the metadata is being passed correctly.